test/search-backend/metadata-search-backend.test.ts

```diff
@@ -81,7 +81,7 @@ describe('MetadataSearchBackend', () => {
         servicePath: '/baz',
         debuggingEnabled: true,
       });
-      await backend.performSearch({ query: 'boop' });
+      await backend.performSearch({ query: 'boop', includeClientUrl: false });
```

Comment:
would prefer this to be not be needed in every call we make unless it's actually being used.

the diff set infers to me that `backend.performSearch` always needs `includeClientUrl` in search param blob, better to gracefully default to falsey.

Reply:
I included this param on a bunch of tests that compare against the full URL, because the testing URL is unpredictable (web-test-runner adds some of its own URL params). But in our real use cases, this param shouldn't need to be passed.

Reply:
right, but if the param is treated as falsey, then I wanna say no test changes would have been needed?

Reply:
also, it's nice to mirror test calls to the real world case that we apply so that we also have a tested validation that our approach is solid..

Reply:
I guess there's a tradeoff here:

- `includeClientUrl` defaults to true (current choice) => we need to update all these tests to exclude it, or to check for it in the expected query string
- `includeClientUrl` defaults to false => no test updates needed, but we need to update every other _real_ caller to include the param explicitly

Reply:
I updated the specificity of the tests so there's no longer a need to have this param on every call.